sql/src/SqlCsv.sk

```diff
@@ -844,6 +846,7 @@ fun replayDiff(
         None()
 
       | None() ->
+        SKDBMonitor.addReplicationReads(1);
```

Comment:
This is not a row to be processed but a table. Will inflate the metric if we're interested in rows. Be nice to capture as a different one so we can see how many different tables are touched in a single txn.